src/test/java/org/minions/devfund/marcos/SquarelotronTest.java

```diff
@@ -0,0 +1,101 @@
+package org.minions.devfund.marcos;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+/**
+ * It is in charged to define the unit tests for Squarelotron class.
+ */
+public class SquarelotronTest {
+
+    /**
+     * Upside Down Flip Basic Test using Squarelotron with dimension 2.
+     */
+    @Test
+    public void upsideDownFlipBasicTest() {
+        final int ringNumber = 1;
+        final int dimension = 2;
+        final Squarelotron squarelotron = new Squarelotron(dimension);
+        final int[][] expectedMatrix = new int[][]{{3, 4}, {1, 2}};
+        Squarelotron result = squarelotron.upsideDownFlip(ringNumber);
+        for (int i = 0; i < expectedMatrix.length; i++) {

```

Comment:
This for is not necessary 
You can declare the assertArrayEquals directly for 2-dimensional arrays
```java
    @Test
    public void testUpsideDownFlipDimension2Ring1() {
        final int ringNumber = 1;
        final int dimension = 2;
        final Squarelotron squarelotron = new Squarelotron(dimension);
        final int[][] expectedMatrix = new int[][]{{3, 4}, {1, 2}};
        Squarelotron result = squarelotron.upsideDownFlip(ringNumber);
        assertArrayEquals(expectedMatrix, result.getCopySquarelotron());
    }
```